backend/src/components/roles.js

```diff
@@ -11,7 +11,9 @@ const roles = {
     //Help functions created in auth module: isValidStudentSearchUserToken, isValidStudentSearchUser
     StudentSearch: [config.get('server:studentSearch:roleAdmin')],
     //Help functions created in auth module: isValidPenRequestBatchUserToken, isValidPenRequestBatchUser
-    PenRequestBatch: [config.get('server:penRequestBatch:roleAdmin')]
+    PenRequestBatch: [config.get('server:penRequestBatch:roleAdmin')],
+    //Help functions created in auth module: isValidStaffAdministrationAdmin
```

Comment:
Should the `isValidStaffAdministrationAdmin` be `isValidStaffAdministrationUser`?